pkg/util/testing/core.go

```diff
@@ -73,3 +74,14 @@ func HasEventAppeared(ctx context.Context, k8sClient client.Client, event corev1
 	}
 	return false, nil
 }
+
+// UseDefaultSecurityContext returns a security context with the default values.
+func UseDefaultSecurityContext() *corev1.SecurityContext {
+	return &corev1.SecurityContext{
+		AllowPrivilegeEscalation: ptr.To(false),
+		Capabilities: &corev1.Capabilities{
+			Drop: []corev1.Capability{"ALL"},
+		},
+		SeccompProfile: &corev1.SeccompProfile{Type: corev1.SeccompProfileTypeRuntimeDefault},
+	}
+}
```

Comment:
We should not make tests programmable as much as possible.
Please follow my comment for the Pod Wrapper.